### ISSUE TITLE
Engine Retries

### ISSUE THREAD

**Issue by ColDog** [battlesnakeio/roadmap#259]
Given that we reverted #258 do we want to try and revisit retries? Do we have enough time to reasonably implement something prior to the tournament?

**Comment by dlsteuer** [comment]
Yes some sort of retry logic should be implemented in the engine. Things to consider when doing this is errors related to timeouts because of retries (i.e. context deadline was exceeded), also make sure to not re-use any existing requests, or snakes will receive empty payloads.
